pkg/cidata/cidata.TEMPLATE.d/lima.env

```
@@ -2,6 +2,7 @@ LIMA_CIDATA_DEBUG={{ .Debug }}
 LIMA_CIDATA_NAME={{ .Name }}
 LIMA_CIDATA_USER={{ .User }}
 LIMA_CIDATA_UID={{ .UID }}
+LIMA_CIDATA_GECOS={{ printf "%q" .GECOS }}
```

There should be no quotes in `lima.env` unless they are part of the value. The file is not sourced, but processed in a read loop:

```console
$ export "GECOS=Jan Dubois"
$ echo $GECOS
Jan Dubois
```

```suggestion
LIMA_CIDATA_GECOS={{ .GECOS }}
```